common/scripted_triggers/wc_scripted_plot_triggers.txt

```diff
@@ -0,0 +1,76 @@
+can_be_killed_by_poisoned_wine_trigger = {
+	can_be_drunkard_trigger = yes
+}
+can_be_killed_by_falling_trigger = {
+	is_humanoid_trigger = yes
+	dark_public_religion_trigger = no
+	
+	can_fly_trigger = no
+}
+can_be_killed_by_archer_ambush_trigger = {
+	#Tribal/nomadic ruler touring in a carriage... sounds weird... don't know why Paradox haven't disabled it
+	is_tribal = no
+	is_nomadic  no
+	
+	is_humanoid_trigger = yes
+	dark_public_religion_trigger = no
+}
+can_be_killed_by_manure_explosion_trigger = {
+	#Tribal/nomadic ruler touring and entering taverns... sounds weird... don't know why Paradox haven't disabled it
+	is_tribal = no
+	is_nomadic  no
+	
+	is_humanoid_trigger = yes
+	dark_public_religion_trigger = no
+}
+can_be_killed_by_infanticide_trigger = {
+	# Everyone? It's the only way to kill a character under 5 years old
+}
+can_be_killed_by_prisoner_break_trigger = {
+	# Everyone? It's the only way to kill a prisoner
+}
+can_be_killed_by_bathtub_drowning_trigger = {
+	is_humanoid_trigger = yes
+	can_breath_underwater_trigger = no
+}
+can_be_killed_by_boating_accident_trigger = {
+	can_breath_underwater_trigger = no
+}
+can_be_killed_by_poisonous_snake_trigger = {
+	is_humanoid_trigger = yes
+	can_get_sick_trigger = yes
+}
+can_be_killed_by_balcony_plummet_trigger = {
```

Comment:
Event speaks about castle balcony, so it can be weird for tribal/nomadic rulers

Reply:
Added tribal, nomad limitations in https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth/pull/1219/commits/0d514b9deee420a8667987c748a9ca7512663f8c.